addons/product_margin/tests/test_product_margin.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import tools
+from odoo.tests import common
+from odoo.modules.module import get_resource_path
+
+
+class TestProductMargin(common.TransactionCase):
+    ''' In order to test the product_margin module '''
+
+    def _load(self, module, *args):
+        tools.convert_file(self.cr, 'account_asset',
+                           get_resource_path(module, *args),
+                           {}, 'init', False, 'test', self.registry._assertion_report)
+
+    def setUp(self):
```

Comment:
Useless setUp, there's a single test method and little of worth in the setup function.